Resources/public/javascript/forms.js

```diff
@@ -1,16 +1,61 @@
+let removePopup = `<div class="sonata-ba-delete popup-file-delete mfp-hide">
```

Comment:
We could you use twig fot handling the translations. But this code is all in JS, so we don't have access to twig here.